config/application.rb

```diff
@@ -67,7 +67,7 @@ class Application < Rails::Application
     config.filter_parameters += [:content, :password, :terms_of_service_non_production]
 
     # Disable dumping schemas after migrations.
```

Comment:
I think this comment should be updated.

Reply:
Thanks for catching that! It looks like `true` is the default, so I've just removed both of these lines